build-bench-env.sh

```diff
@@ -663,6 +670,21 @@ if test "$setup_redis" = "1"; then
   popd
 fi
 
+if test "$setup_rocksdb" = "1"; then
+  phase "build rocksdb $version_rocksdb"
+
+  pushd $devdir
+  if test -d "rocksdb"; then
+    echo "$devdir/rocksdb already exists; no need to download it"
+  else
+    git clone https://github.com/facebook/rocksdb.git
+  fi
+  cd rocksdb
+  git checkout "$version_rocksdb"
+  DISABLE_JEMALLOC=1 make all -j $procs
```

Comment:
Don't build the whole project, only what's required :)

Reply:
Done